src/components/global/Task/Task.jsx

```diff
@@ -1,10 +1,19 @@
 import React, { useState } from "react";
 import RightChevron from "../../../assets/icons/RightChevron";
 import DarkModeContext from "../../../services/theme-context";
+import NewSubtask from "../../NewSubtask";
 import Subtask from "./Subtask";
 
 function Task(props) {
-  const { name, checked, expanded, subtasks, selected, onClick } = props;
+  const {
+    name,
+    checked,
+    expanded,
+    subtasks,
+    selected,
+    onClick,
+    taskId,
+  } = props;
```

Comment:
Hey can you please rebase from main-group2 - I am concerned that there are a bunch of props missing here